simplified-viewer-epub-readium2/src/main/java/org/nypl/simplified/viewer/epub/readium2/ReaderViewerR2.kt

```diff
@@ -32,7 +33,11 @@ class ReaderViewerR2 : ViewerProviderType {
       is BookFormat.BookFormatAudioBook ->
         false
       is BookFormat.BookFormatEPUB ->
-        format.adobeRights == null
+        when (format.drmInformation) {
+          is BookDRMInformation.ACS -> false
+          is BookDRMInformation.LCP -> true
+          BookDRMInformation.None -> true
```

Comment:
For better or worse, Kotlin is whitespace-sensitive in places. The lack of `is` in one of the clauses is because `None` is an `object` instead of a `data class`.